src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/ConnectionException.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+public class ConnectionException extends RuntimeException {
```

Comment:
Could we rename it to specific connection exception for e.g QueueConnectionException or ServiceBusConnectionException ?

Reply:
It's generic enough to be reused with other connections in case we have to create them, so I would not rename it to something specific which it is not.